src/dftd4.f90

```diff
@@ -16,9 +16,9 @@
 
 module dftd4
    use mctc_io, only : structure_type, new
+   use mctc_ncoord, only : new_ncoord, ncoord_type
    use dftd4_cutoff, only : realspace_cutoff, get_lattice_points
    use dftd4_disp, only : get_dispersion, get_properties, get_pairwise_dispersion
-   use dftd4_ncoord, only : get_coordination_number
```

Comment:
Please don't remove features from the public API. Let's reimplement `get_coordination_number` instead of replacing it.

Reply:
We already broke the API with the recent D4S implementation, i.e., we need to make a major release next anyway. Should we still retain `get_coordination_number`?

Reply:
Yes, let's keep it, if only for documentation purpose. The new ncoord type offers much flexibility in the initialisation, having one place in the library defining the usage is better then having several places in the code base and documentation.